src/agent/debuglet.js

```diff
@@ -146,8 +146,10 @@ Debuglet.normalizeConfig_ = function(config) {
   var envConfig = {
     logLevel: process.env.GCLOUD_DEBUG_LOGLEVEL,
     serviceContext: {
-      service: process.env.GAE_SERVICE || process.env.GAE_MODULE_NAME,
-      version: process.env.GAE_VERSION || process.env.GAE_MODULE_VERSION,
+      service: process.env.GAE_SERVICE || process.env.GAE_MODULE_NAME ||
```

Comment:
Is there a known situation where `GAE_SERVICE` or `GAE_MODULE_NAME` are intentionally set to the empty string.  I don't think this is the case, but I just want to make sure.